app/views/dashboard/epic_queues/_epic_queue_records_table.html.haml

```diff
@@ -18,22 +18,22 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-.bootstrap-table-dropdown-overflow
+.card
   #epic-queues-custom-toolbar
   %table.epic-queue-records-table{ data: { toggle: 'table', search: 'true', 'show-columns' => 'true', 'show-refresh' => 'true', 'show-toggle' => 'true', 'show-export' => 'true', pagination: 'true', side_pagination: 'server', url: dashboard_epic_queue_records_path, striped: 'true', toolbar: '#epic-queues-custom-toolbar' } }
     %thead.primary-header
```

Comment:
Can you replace the `primary-header` class with `bg-light`? The `<theme-color>-header` classes no longer exist and we're now using `bg-light` to style theads